libra-swarm/src/main.rs

```diff
@@ -40,21 +40,21 @@ fn main() {
     libra_logger::init_for_e2e_testing();
 
     let mut validator_swarm = LibraSwarm::configure_swarm(
+        None, /* template config */
         num_nodes,
         RoleType::Validator,
         args.config_dir.clone(),
-        None, /* template_path */
         None, /* upstream_config_dir */
     )
     .expect("Failed to configure validator swarm");
 
     let mut full_node_swarm = if num_full_nodes > 0 {
         Some(
             LibraSwarm::configure_swarm(
+                None, /* template config */
                 num_full_nodes,
                 RoleType::FullNode,
                 None, /* config dir */
-                None, /* template_path */
```

Comment:
why did we move the position of the parameter? typically optional parameters are placed at the end.